styleguide/js/zoolander.js

```diff
@@ -44,7 +73,7 @@ jQuery(document).ready(function($) {
         .on('touchend', dropDownTrigger);
   } else {
     $(".navbar-tertiary-dropDownTrigger")
-      .hover(dropDownTrigger)
+      .on('click', dropDownTrigger)
       .on('click', function (e) { e.stopPropagation(); });
   }
```

Comment:
If you're going to be changing JavaScript here, this is fine, but it's gonna a `www` PR as well:
https://github.rackspace.com/rswebteam/www/blob/master/styleguide/js/src/rackspace.js#L202